debian/control

```diff
@@ -30,7 +30,7 @@ Depends:
   ${misc:Depends},
   ${python3:Depends},
   ${shlibs:Depends}
-Suggests: python3-revpi-device-info
+Suggests: revpi-sos-report
```

Comment:
Is it really a Suggests? Shouldn't we use Recommends?
We used Suggests only to be able to add the weak dependency during buster.

Reply:
I'd say it is a suggestion because revpi-tools works perfectly fine without the sos-report. It's more like a support / debug tool.